### PR TITLE
extend config object

### PR DESCRIPTION
**Pull request by akash1810**
to make use of the config injected by the page, this includes stuff like:
- ophan id
- ga id
- currently running tests

### CODE REVIEW

File: src/js/boot.js.tpl [hunk 1]
```diff
@@ -15,14 +15,16 @@ define([], function() {
             // Loading message while we fetch JS / CSS
             el.innerHTML = '<div style="font-size: 24px; text-align: center; padding: 72px 0; font-family: \'Guardian Egyptian Web\',Georgia,serif;">Loading…</div>';
 
-            config = {
-                'assetPath': '<%= assetPath %>',
+            const extraConfig = {
+                'assetPath': '',
```
Comment (gidsg) [inline src/js/boot.js.tpl]: Is the idea that this value will still be populated? I think that variable makes it based on the deployed path in the interactive bucket.

Comment (akash1810) [inline src/js/boot.js.tpl]: As far as I can tell, `config` is injected by [frontend](https://github.com/guardian/frontend/blob/master/static/src/javascripts/bootstraps/standard/main.js#L132-L140)(namely line [137](https://github.com/guardian/frontend/blob/master/static/src/javascripts/bootstraps/standard/main.js#L137))

Before, we were overriding it 😊. Now we extend it if its set, or set it to `extraConfig` if its not set.

Comment (akash1810) [inline src/js/boot.js.tpl]: latest commit makes the code clearer and cleaner now

**Comment by akash1810** [comment]
Adding a do not merge label to this.

The main reason for this work is to add GA tracking so will unblock it once that PR is done to keep things simple and reduce risk.

The main reason for adding GA tracking is because we get graphs/dashboards out of the box with GA; this'll make reporting stats to Editorial easier/self-serve.

**Comment by akash1810** [comment]
Superseded by #88. Closing.

**Pull request closed without merging**